src/api/app/controllers/webui/repositories_controller.rb

```diff
@@ -148,14 +148,16 @@ def create_dod_repository
   # POST project/create_image_repository
   def create_image_repository
     authorize @project, :update?
-    repository = @project.repositories.create!(name: 'images')
-    Architecture.available.each do |architecture|
-      repository.repository_architectures.create!(architecture: architecture)
-    end
-    @project.prepend_kiwi_config
+    repository = @project.repositories.new(name: 'images')
 
     if repository.save
```

Comment:
The save is not really used as there are the ``bang`` methods used, no?

Reply:
Yes, you're right - it wasn't used before, but now it's required.